release.nix

```diff
@@ -5,6 +5,13 @@
   changelog = {
     # Find the current version number with `git log --pretty=%h | wc -l`
     entries = [
+      {
+        version = 548;
+        changes = ''
+          Add $IN_LORRI_SHELL environment variable to allow differentiation
+          between `nix-shell` and `lorri shell`.
```

Comment:
We should mention that it carries the path to the evaluated shell file as value.